cylc/flow/scheduler.py

```diff
@@ -1225,8 +1225,7 @@ def release_queued_tasks(self) -> bool:
 
         """
         if (
-            not self.is_paused
-            and self.stop_mode is None
+            self.stop_mode is None
```

Comment:
> there's nothing wrong with defining workflow pause as pausing all **automatic** job submission

This has gone a little further than that one word change above as we are now continuing to process the queues whilst the workflow is running.

I don't think we actually need to process the entire queue to achieve this, we just need to skip any manually triggered tasks through the queue.

If we pass `self.is_paused` through to `release_queued_tasks` we could reduce the scope of this override such that it only impacts manually triggered tasks.